fdbserver/DiskQueue.actor.cpp

```diff
@@ -594,6 +597,8 @@ class RawDiskQueue_TwoFiles : public Tracked<RawDiskQueue_TwoFiles> {
 	}
 
 	ACTOR static void shutdown(RawDiskQueue_TwoFiles* self, bool deleteFiles) {
+		state TrackMe trackMe(self);
```

Comment:
Having a TrackMe in shutdown would prevent shutdown forever because onSafeToDestruct() is waiting for all of the TrackMe's to be destructed.